src/components/join-flow/join-flow-steps.jsx

```diff
@@ -187,11 +190,143 @@ class UsernameStep extends React.Component {
+
+/*
+ * BirthDateStep
+ */
+
+const getBirthMonthOptions = intl => ([
+    {value: 'null', label: intl.formatMessage({id: 'general.month'})},
+    {value: '1', label: intl.formatMessage({id: 'general.monthJanuary'})},
+    {value: '2', label: intl.formatMessage({id: 'general.monthFebruary'})},
+    {value: '3', label: intl.formatMessage({id: 'general.monthMarch'})},
+    {value: '4', label: intl.formatMessage({id: 'general.monthApril'})},
+    {value: '5', label: intl.formatMessage({id: 'general.monthMay'})},
+    {value: '6', label: intl.formatMessage({id: 'general.monthJune'})},
+    {value: '7', label: intl.formatMessage({id: 'general.monthJuly'})},
+    {value: '8', label: intl.formatMessage({id: 'general.monthAugust'})},
+    {value: '9', label: intl.formatMessage({id: 'general.monthSeptember'})},
+    {value: '10', label: intl.formatMessage({id: 'general.monthOctober'})},
+    {value: '11', label: intl.formatMessage({id: 'general.monthNovember'})},
+    {value: '12', label: intl.formatMessage({id: 'general.monthDecember'})}
+]);
+const curYearRaw = (new Date()).getYear();
+const curYear = curYearRaw + 1900;
+const birthYearOptions = Array(curYearRaw + 2).fill()
+    .map((_, i) => (
+        {value: String(curYear + 1 - i), label: String(curYear + 1 - i)}
+    ));
+
+class BirthDateStep extends React.Component {
+    constructor (props) {
+        super(props);
+        bindAll(this, [
+            'handleValidSubmit',
+            'validateForm',
+            'validateSelect'
+        ]);
+    }
+    validateSelect (selection) {
+        if (selection === 'null') {
+            return this.props.intl.formatMessage({id: 'form.validationRequired'});
+        }
+        return null;
+    }
+    validateForm () {
+        return {};
+    }
+    handleValidSubmit (formData, formikBag) {
+        formikBag.setSubmitting(false);
+        this.props.onNextStep(formData);
+    }
+    render () {
+        const birthMonthOptions = getBirthMonthOptions(this.props.intl);
+        birthYearOptions[0] = {
+            value: 'null',
+            label: this.props.intl.formatMessage({id: 'general.year'})
+        };
+        return (
+            <Formik
+                initialValues={{
+                    birth_month: 'null',
+                    birth_year: 'null'
+                }}
+                validate={this.validateForm}
+                validateOnBlur={false}
+                validateOnChange={false}
+                onSubmit={this.handleValidSubmit}
+            >
+                {props => {
+                    const {
+                        errors,
+                        handleSubmit,
+                        isSubmitting
+                    } = props;
+                    return (
+                        <JoinFlowStep
+                            description={this.props.intl.formatMessage({id: 'registration.birthDateStepDescription'})}
+                            headerImgSrc="/images/hoc/getting-started.jpg"
```

Comment:
is this an image that exists already? (Either way is fine with me since I know this isn't final, but just curious)

Reply:
yes, just using as a placeholder